src/lksearch/TESSSearch.py

```diff
@@ -287,76 +289,26 @@ def _sort_TESS(self):
         )
         self.table = df
 
+    @deprecated(
+        "1.0.1",
+        warning_type=SearchDeprecationWarning,
+        message="The {func} {obj_type} has been deprecated as astroquery no longer supports querying individual FFI files.",
+    )
     def search_sector_ffis(
         self,
         # tmin: Union[float, Time, tuple] = None,
         # tmax: Union[float, Time, tuple] = None,
         # search_radius: Union[float, u.Quantity] = 0.0001 * u.arcsec,
         # exptime: Union[str, int, tuple] = (0, 9999),
-        sector: Union[int, type[None]],  # = None,
-        **extra_query_criteria,
+        # sector: Union[int, type[None]],  # = None,
```

Comment:
As mentioned in the tests, removing the arguments entirely raises `TypeError: search_sector_ffis() takes 1 positional argument but 2 were given` instead of the deprecation error/warning. I think you should keep the arguments as-is so that users and prior-written code will raise the appropriate error.